zrcs/zrc-2.md

```diff
@@ -0,0 +1,372 @@
+| --------- | ------------- | -------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| eventName | `SendSuccess` | Sending is successful.     | `from`: `ByStr20`, `recipient`: `ByStr20`, and `amount`: `Uint128`.                             |
+| eventName | `Error`       | Sending is not successful. | TBD. |
+
+#### 2. OperatorSend
+
+```ocaml
+(* @dev: Moves amount tokens from sender to recipient. The caller must be an operator of sender. *)
+(* @param sender:     Address of the sender whose balance is decreased.                          *)
+(* @param recipient:  Address of the recipient whose balance is increased.                       *)
+(* @param amount:     Amount of tokens to be sent.                                               *)
+transition OperatorSend(from: ByStr20, to: ByStr20, amount: Uint128)
+```
+
+|        | Name        | Type      | Description                                          |
+| ------ | ----------- | --------- | ---------------------------------------------------- |
+| @param | `sender`    | `ByStr20` | Address of the sender whose balance is decreased.    |
+| @param | `recipient` | `ByStr20` | Address of the recipient whose balance is increased. |
+| @param | `amount`    | `Uint128` | Amount of tokens to be sent.                         |
+
+|           | Name          | Description                | Event Parameters                                                                                                                                                                                                                   |
+| --------- | ------------- | -------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| eventName | `OperatorSendSuccess` | Sending is successful.     | `from`: `ByStr20`, `to`: `ByStr20`, and `amount`: `Uint128`.                             |
+| eventName | `Error`       | Sending is not successful. | - emit `CodeNotAuthorised` if the transition is called by a user who is not an approved operator. |
+
+#### 3. Burn
+
+```ocaml
+(* @dev: Burn existing tokens. Only tokenOwner or approved operator can burn a token *)
+(* @param from:                             Address holding the tokens to be burned. *)
+(* @param amount:                           Number of tokens to be destroyed.        *)
+transition Burn(from: ByStr20, amount: Uint128)
+```
+
+|        | Name      | Type      | Description                              |
+| ------ | --------- | --------- | ---------------------------------------- |
+| @param | `from`    | `ByStr20` | Address holding the tokens to be burned. |
```

Comment:
maybe `from` can be renamed to be `burnee` or `burn_account`.

Reply:
burn_account it is!